### PR TITLE
chore: publish v1.0.0-dev.1

### PR DESCRIPTION
**Pull request by dshukertjr**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## [1.0.0-dev.1]
+- BREAKING: update supabase package [v1.0.0-dev.1](https://github.com/supabase-community/supabase-dart/blob/main/CHANGELOG.md#100-dev1)
```
Comment (bdlukaa) [inline CHANGELOG.md:2]: https://github.com/supabase-community/supabase-flutter/pull/136
https://github.com/supabase-community/supabase-flutter/pull/124
https://github.com/supabase-community/supabase-flutter/pull/131

these should be added

**Review by bdlukaa — CHANGES_REQUESTED** [review]

**Comment by dshukertjr** [comment]
Ah, tests...

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## [1.0.0-dev.1]
+- feat: add Mac OS and Windows support for deeplinks
+- BREAKING: Remove SupabaseAuthRequiredState as well as overriding methods in SupabaseAuthState
```
Comment (bdlukaa) [inline CHANGELOG.md:3]: Migration guide for breaking change

**Review by bdlukaa — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (dshukertjr) [reply CHANGELOG.md:3]: Great call! Adding it now!

**Review by bdlukaa — APPROVED** [review]

**Pull request merged**